user_docs/en/userGuide.t2t

```diff
@@ -837,6 +837,15 @@ Usually, NVDA raises the pitch slightly for any capital letter, but some synthes
 If this checkbox is checked, NVDA will make a small beep each time it encounters a capitalized character by itself.
 Like the "say cap for capitals" checkbox, this is useful for Synthesizers that can't change their pitch for capital letters.
 
+==== Speak Numbers as ====
+This combo box allows you to choose whether nvda will speak numbers as words, single digits, Double digits, or Triple digits. 
+For example, each of the settings and the result will be shown in a table below for the number 54321.
+|| option | reads as |
+| Words | fifty four thousand three hundred twenty 1 |

```

Comment:
I think we should note that the exact pronunciation of numbers with the Words option (or whatever it gets renamed to) is dependent on the synthesizer being used. You could just add a sentence below the table to explain this.